project-base/storefront/helpers/pagination/parseLoadMoreFromQuery.ts

```diff
@@ -0,0 +1,4 @@
+export const parseLoadMoreFromQuery = (query: string | string[] | undefined): number => {
+    const parsedLoadMore = Number(query);
+    return isNaN(parsedLoadMore) ? 0 : parsedLoadMore;
+};
```

Comment:
I guess this is mainly for remove undefined value and replace it with `0`. But cannot we simply use `undefined` in case you don't have loadmore?

Reply:
We apply arithmetic operations to the result of this function, so I would rather not.